src/app/components/gameOverDialog.ts

```diff
@@ -0,0 +1,64 @@
+import { Game } from '../game'
+import { createElement, secondsToDate } from '../utils'
+import { renderPopup } from './popup'
+
+export default class GameOverDialog {
```

Comment:
we can use Dialog or Modal, in project I changed every popup to Modal. You can change them to Dialogs or change this on to gameOverModal